### ISSUE TITLE
ItemBlacklister - Choice menu below artifacts not listed?

### ISSUE THREAD

**Issue by AkitaAttribute** [RyanPallesen/FloodWarning#5]
Hoping you're interested in fixing this.  I know this hasn't been touched in 4 years, but it's a mod that is interesting to me.  

Thanks.

**Comment by Darkmega18** [comment]
indeed, seems the mod has ceased working in recent months. :< If you're still interesting in risk of rain, please consider updating it.
